dymos/phase/phase.py

```diff
@@ -1216,13 +1218,17 @@ def _add_timeseries_output(self, name, output_name=None, units=_unspecified, sha
         if timeseries not in self._timeseries:
             raise ValueError(f'Timeseries {timeseries} does not exist in phase {self.pathname}')
 
-        if name not in self._timeseries[timeseries]['outputs']:
-            self._timeseries[timeseries]['outputs'][name] = {}
-            self._timeseries[timeseries]['outputs'][name]['output_name'] = output_name
-            self._timeseries[timeseries]['outputs'][name]['wildcard_units'] = {}
+        if output_name not in self._timeseries[timeseries]['outputs']:
```

Comment:
I might be confused, but will `output_name` ever be in `self._timeseries[timeseries]['outputs']` ? (which is a list of `TimeseriesOutputOptionsDictionary` objects).

Reply:
Fixed this too, and added a test that covers the case of adding constraints/timeseries that have already been added as timeseries.